src/common/legacy_config_opts.h

```diff
@@ -1548,3 +1548,6 @@ OPTION(rgw_sts_key, OPT_STR)
 OPTION(rgw_s3_auth_use_sts, OPT_BOOL)  // should we try to use sts for s3?
 OPTION(rgw_sts_max_session_duration, OPT_U64) // Max duration in seconds for which the session token is valid.
 OPTION(fake_statfs_for_testing, OPT_INT) // Set a value for kb and compute kb_used from total of num_bytes
+OPTION(rgw_sts_token_introspection_url, OPT_STR)  // url for introspecting web tokens
+OPTION(rgw_sts_client_id, OPT_STR) // Client Id
+OPTION(rgw_sts_client_secret, OPT_STR) // Client Secret
```

Comment:
We don't actually have to put things in here. It's for legacy options that already have users so their code doesn't all have to be rewritten.

(We don't have to NOT put things in here either, though. No need to take it out, just a note.)